recipes/colour-science/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "colour-science" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "1234e3c8d4ac120030b605cbe1c578cb4dc2a9b9927a36efdd1bcd424c723290" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy >=0.16.0
+
+  run:
+    - matplotlib >=1.3.1
+    - mock
+    - python
+    - scipy >=0.16.0
+    - six >=1.10.0
+
+test:
+  imports:
+    - colour
+    - colour.adaptation
+    - colour.adaptation.dataset
+    - colour.adaptation.tests
+    - colour.algebra
+    - colour.algebra.coordinates
+    - colour.algebra.tests
+    - colour.algebra.tests.tests_coordinates
+    - colour.appearance
+    - colour.appearance.tests
+    - colour.characterisation
+    - colour.characterisation.dataset
+    - colour.characterisation.dataset.colour_checkers
+    - colour.characterisation.dataset.displays
+    - colour.characterisation.dataset.displays.crt
+    - colour.characterisation.dataset.displays.lcd
+    - colour.characterisation.tests
+    - colour.colorimetry
+    - colour.colorimetry.dataset
+    - colour.colorimetry.dataset.illuminants
+    - colour.colorimetry.dataset.light_sources
+    - colour.colorimetry.tests
+    - colour.constants
+    - colour.corresponding
+    - colour.corresponding.dataset
+    - colour.corresponding.tests
+    - colour.difference
+    - colour.difference.tests
+    - colour.io
+    - colour.io.tests
+    - colour.models
+    - colour.models.dataset
+    - colour.models.rgb
+    - colour.models.rgb.dataset
+    - colour.models.rgb.tests
+    - colour.models.rgb.transfer_functions
+    - colour.models.rgb.transfer_functions.tests
+    - colour.models.tests
+    - colour.notation
+    - colour.notation.dataset
+    - colour.notation.dataset.munsell
+    - colour.notation.tests
+    - colour.phenomenons
+    - colour.phenomenons.tests
+    - colour.plotting
+    - colour.plotting.dataset
+    - colour.plotting.tests
+    - colour.quality
+    - colour.quality.dataset
+    - colour.quality.tests
+    - colour.recovery
+    - colour.recovery.dataset
+    - colour.recovery.tests
+    - colour.temperature
+    - colour.temperature.tests
+    - colour.utilities
+    - colour.utilities.tests
+    - colour.volume
+    - colour.volume.dataset
+    - colour.volume.tests
+
+about:
+  home: http://colour-science.org/
+  license: BSD 3-Clause
+  license_family: BSD
+  summary: 'Colour Science for Python'
+  doc_url: http://colour.readthedocs.io/
+  dev_url: http://github.com/colour-science/colour
+
+extra:
+  recipe-maintainers:
+    - KelSolaar
+    - MichaelMauderer
```

Comment:
@michaelMauderer are you ok being added as a co-maintainer of this conda-forge recipe?